README.md

```diff
@@ -1,132 +1,15 @@
-# Getting started with ASP.NET on Bluemix
-This guide will take you through the steps to get started on Bluemix with the help of a sample application. In 10 minutes or less, you’ll learn to:
-- Set up a development environment
-- Download sample code
-- Run the application locally
-- Run the application on Bluemix Cloud Foundry
-- Add a Bluemix Database service
-- Connect to the database from your local application
+#ASP.NET getting started application
+The Bluemix Getting started tutorial for ASP.NET uses this sample application to provide you with a sample workflow for working with any Dotnet app on Bluemix; you set up a development environment, deploy an app locally and on Bluemix, and integrate a Bluemix database service in your app.
 
+The ASP.NET getting started application uses a[Cloudant noSQL DB service](https://console.bluemix.net/catalog/services/cloudant-nosql-db) to add information to a database and then return information from a database to the UI. To learn more about how the app connects to Cloudant, see the [Cloudant library for Node.js](https://www.npmjs.com/package/cloudant).
```

Comment:
This sample doesn't use Cloudant, it uses the ClearDB MySQL service from the Bluemix catalog and the [SapientGuardian](https://github.com/SapientGuardian/SapientGuardian.EntityFrameworkCore.MySql) provider for Entity Framework Core.  Although, we should probably look at switching this to [Pomelo](https://docs.microsoft.com/en-us/ef/core/providers/pomelo/) instead of SapientGuardian.
@opiethehokie ^